src/sdk/conference/client.js

```diff
@@ -115,6 +115,7 @@ export const ConferenceClient = function(config, signalingImpl) {
   const participants = new Map(); // Key is participant ID, value is a Participant object.
   const publishChannels = new Map(); // Key is MediaStream's ID, value is pc channel.
   const channels = new Map(); // Key is channel's internal ID, value is channel.
+  let mainChannel = null;
```

Comment:
When all publications and subscriptions are ended, will you close the channel and create a new one for new publish/subscribe requests, or leave it as an empty channel and use it again for new publish/subscribe requests?